ome/omero-figure
# Scalebar units

Allow user to choose units for scalebar - Fixes #331

Testing can use images with various pixel sizes created at 
https://merge-ci.openmicroscopy.org/web/webclient/?show=dataset-14769
(created with this script:
https://gist.github.com/will-moore/68cccecf9646571fa1d96a981eac0cce)

<img width="386" alt="Screenshot 2020-01-29 at 20 43 33" src="https://user-images.githubusercontent.com/900055/73395800-1b808c80-42d8-11ea-8ad0-8be147b72f22.png">


To test:
 - WITHOUT this PR, create a figure with containing images with various pixel sizes (see Dataset above), 
     - Add scalebars with labels, using same Units as the pixel size units.
     - Leave 1 or 2 images without scalebars.
     - Export to PDF/TIFF to have a record of what it looked like.
 - Update to use this PR - open the figure created earlier and test that scalebars look the same.
 - Change the units on existing scalebars - they should update as expected.
 - Add new images and add scalebars to these and to older images without scalebars: Default scalebars should be created in the same units as pixel sizes.
 - Try changing units for scalebar... And show label...
 - Label should update & scalebar should be expected length
 - Test export as TIFF and PDF
